frontend/src/extensions/external-lib-app-extensions/plantuml/plantuml-not-configured-alert.tsx

```diff
@@ -1,20 +1,17 @@
 /*
- * SPDX-FileCopyrightText: 2022 The HedgeDoc developers (see AUTHORS file)
+ * SPDX-FileCopyrightText: 2023 The HedgeDoc developers (see AUTHORS file)
  *
  * SPDX-License-Identifier: AGPL-3.0-only
  */
 import React from 'react'
-import { Trans, useTranslation } from 'react-i18next'
+import { useTranslation } from 'react-i18next'
+import { ErrorBox } from '../../../components/common/error/error-box'
 
 /**
  * Renders an alert if plantuml is not configured.
  */
 export const PlantumlNotConfiguredAlert: React.FC = () => {
-  useTranslation()
+  const { t } = useTranslation()
 
-  return (
-    <p className='alert alert-danger'>
-      <Trans i18nKey={'renderer.plantuml.notConfigured'} />
-    </p>
-  )
+  return <ErrorBox error={t('renderer.plantuml.notConfigured')} />
```

Comment:
How about allowing `ErrorBox` to take a translation key directly as parameter? We do this in other places as well.